tracer/src/Datadog.Trace/ClrProfiler/InstrumentationCategory.cs

```diff
@@ -7,9 +7,9 @@ namespace Datadog.Trace.ClrProfiler
 {
     internal enum InstrumentationCategory
     {
-        [DefinitionsId("FFAFA5168C4F4718B40CA8788875C2DA", "61BF627FA9B5477F85595A9F0D68B29C")]
+        [DefinitionsId("FFAFA5168C4F4718B40CA8788875C2DA", "61BF627FA9B5477F85595A9F0D68B29C", "6410E14A2A2343BABBB45940190E1C3F")]
```

Comment:
Basically each time we invoke the profiler to register/unregister instrumentations we attach a unique GUID to the set of definitions so we don't double-register them. Since I added a new profiler method to register interface instrumentations, I needed to generate new GUID's for the new set of instrumentations.

And the new GUID's I added were just newly generated GUID's, with no significance other than they don't match any of our existing GUID's 😆